code/modules/power/tesla/energy_ball.dm

```diff
@@ -33,6 +33,7 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 	var/produced_power
 	var/energy_to_raise = 32
 	var/energy_to_lower = -20
+	var/home = null // Syndie Beacon target
```

Comment:
Energy Ball has its own target obj, so I wanted to use home for clarity. I have no idea what gets inherited.